bentoml/_internal/frameworks/detectron.py

```diff
@@ -169,16 +169,17 @@ def save(
     """  # noqa
 
     context: t.Dict[str, t.Any] = {
-        "detectron2": _detectron2_version,
-        "torch": _torch_version,
+        "framework": "detectron2",
+        "detectron2_version": _detectron2_version,
```

Comment:
this way it may be hard for us to find the corresponding package name, let's make it:
```
context = {
   "framework_name": "detectron2",
   "pip_dependencies": [f"detectron2=={_detectron2_version}", f"torch=={__torch_version}"]
}
```